web/src/components/layout/header/Navbar.vue

```diff
@@ -24,13 +24,6 @@
     </div>
     <!-- Right Icons Box -->
     <div class="flex ml-auto -m-1.5 items-center space-x-2">
-      <!-- Dark Mode Toggle -->
-      <IconButton
-        :icon="darkMode ? 'dark' : 'light'"
-        :title="$t(darkMode ? 'color_scheme_dark' : 'color_scheme_light')"
-        class="navbar-icon"
-        @click="darkMode = !darkMode"
-      />
```

Comment:
What do you think about adding an `A` for automatic mode? Its sometimes pretty nice to quickly switch the mode especially as chromium is still having this bug that it does not reflect the OS setting 🙈